screens/CheckoutScreen.js

```diff
@@ -181,61 +184,113 @@ export default class CheckoutScreen extends React.Component {
+        <View style={{ display: 'flex', flexDirection: 'row', flex: 1 }}>
           {/* Display products */}
-          <ProductsContainer>
-            {Object.entries(cart).map(([id, product]) => (
-              <QuantityModal key={id} product={product} isLineItem={false} callback={this.updateQuantityCallback} />
-            ))}
-          </ProductsContainer>
+          <View style={{ display: 'flex', flex: 5, flexDirection: 'column' }}>
+            <ProductsContainer
+              ref={scrollView => {
+                this._scrollView = scrollView;
+              }}>
+              {Object.entries(cart).map(([id, product]) => (
+                <QuantityModal key={id} product={product} isLineItem={false} callback={this.updateQuantityCallback} />
+              ))}
+            </ProductsContainer>
+            <BottomBar style={{ display: 'flex', flexDirection: 'row' }}>
+              {this.alphabeticalScrollTab('A-K', 'A')}
+              {this.alphabeticalScrollTab('L-S', 'L')}
+              {this.alphabeticalScrollTab('T-Z', 'T')}
+            </BottomBar>
+          </View>
           {/* Right column */}
           <SaleContainer>
-            <Subhead>Current Sale</Subhead>
-            {/* Cart container */}
-            <View style={{ height: '40%', paddingBottom: '5%' }}>
-              <ScrollView>
-                {Object.entries(cart).map(([id, product]) => {
-                  return (
-                    product.quantity > 0 && (
-                      <QuantityModal key={id} product={product} isLineItem callback={this.updateQuantityCallback} />
-                    )
-                  );
-                })}
-              </ScrollView>
-            </View>
-            <RewardModal
-              totalBalance={totalBalance}
-              customer={customer}
-              rewardsAvailable={this.state.rewardsAvailable}
-              rewardsApplied={this.state.rewardsApplied}
-              callback={this.applyRewardsCallback}
-            />
-            <Text
+            <View
               style={{
-                fontWeight: 'bold',
-                textAlign: 'center'
+                paddingTop: 13,
+                paddingLeft: 14,
+                paddingRight: 14,
+                paddingBottom: 0,
+                flexDirection: 'column',
+                justifyContent: 'space-between',
+                flex: 1
               }}>
-              Order Total ${totalSale.toFixed(2)}
-            </Text>
-            <TouchableOpacity onPress={() => this.handleSubmit()}>
-              <TextHeader style={{ color: '#008550' }}>Checkout</TextHeader>
-            </TouchableOpacity>
+              <View style={{ height: '60%' }}>
+                <Subhead>Current Sale</Subhead>
+                {/* Cart container */}
+                <View style={{ height: '100%', paddingBottom: '5%' }}>
+                  <ScrollView>
```

Comment:
UX detail: Can you add something where it scrolls to the bottom of the ScrollView on size change (aka whenever a new product is added to this list)? Otherwise, it lacks that kind of confirmation that the product was added and could cause some confusion.
Something like this: https://stackoverflow.com/a/42736127